lips/lip-8.md

```diff
@@ -125,12 +141,65 @@ The structure in this object can be a full pre-approval or just the fields of an
 }
 ```
 
+### ScopeObject
+
+In this object the initiator VASP declares its intent for the pre-approval, this can by one of two options:
+  1. Save the consumer sub-account for future transactions (save_sub_account)- this will enable the initiator VASP (merchant) to charge the sub-account in the future but will require the owner to approve the transaction. When using this option, amount limits are not required
+  2. Save the consumer sub-account and get a consent for future payments (consent) - this enables the initiator VASP (merchant) to charge the sub-account without any interaction with the owner. 
+Also, the scope limits the FundPullPreApprovalObject to certain parameters of time and amount. this object can be changed by the initiator VASP if needed, but any change requires the target VASP to approve the scope change.
+
+
+| Field         | Type  | Required?     | Description   |
+|-------        |------ |-----------    |-------------  |
+| Type | str enum | Y | This can be either save_sub_account or consent |
+| expiration_timestamp | uint | Y | Unix timestamp indicating the time at which this pre-approval will expire - after which no funds pull can occur.  To expire an existing pre-approval early, this field can be updated with the current Unix timestamp. |
+| max_cumulative_amount | [ScopedCumulativeAmountObject](#scopedcumulativeamountobject) | N | Max cumulative amount that is approved for funds pre-approval.  This is the sum across all transactions that occur while utilizing this pre-approval. |
+| max_transaction_amount | [CurrencyObject](#currencyobject) | N | Max transaction amount that is approved for funds pre-approval.  This is the maximum transaction that occurs while utilizing this funds pre-approval. |
+
+```
+{
+    "scope": {
+        "type": "consent",
+        "expiration_timestamp": 72322,
+        "max_transaction_amount": {
+            "amount": 100,
+            "currency": "LBR",
+        }
+    }
+}
+```
+
+### ScopedCumulativeAmountObject
+
+This object describes the scope of an amount
+
+| Field         | Type       | Required?    | Description   |
+|-------        |--------    |-----------   |-------------  |
+| unit          | str enum   | N            | One of: "week", "month", "year" |
```

Comment:
Should we add "day"?  And what about something for an all-time limit - or should a high number of years suffice for that?

Reply:
for "days" - you're right. I will add it. 
for "all-time" - I would go with the latter to keep the structure the same for all options (if we set a unit value of "all-time" then we don't need a "value" field which makes parsing uglier). not crucial, just a matter of taste.